spring-integration-http/src/test/java/org/springframework/integration/http/management/IntegrationGraphControllerParserTests-context.xml

```diff
@@ -12,12 +12,6 @@
 
 	<mvc:annotation-driven />
 
-	<mvc:cors>
-		<mvc:mapping path="/foo"
-					 allowed-origins="http://foo.bar.com"
-					 allowed-methods="GET" />
-	</mvc:cors>
-
-	<graph-controller path="/foo" />
+	<graph-controller path="/foo" allowed-origins="http://foo.bar.com" />
```

Comment:
```
for (HandlerInterceptor handlerInterceptor : executionChain.getInterceptors()) {
	// Assert the CORS config
	assertTrue(handlerInterceptor.preHandle(request, response, handler));
}

handlerAdapter.handle(request, response, handler);
```
WDYT ?

But that doesn't help us to overcome  the XML-annotations misconfiguration... :cry: